internal/services/ticker/cron.go

```diff
@@ -95,6 +95,7 @@ func (t *TickerImpl) handleScheduleCron(ctx context.Context, cron *dbsqlc.PollCr
 	)
 
 	if err != nil {
+		t.repo.Workflow().DeleteInvalidCron(ctx, cron.ID)
```

Comment:
handle error and make sure it's logged?

Reply:
also: should we be pretty sure that this is a cron validation error, not some other random error? perhaps validation errors get silently removed, othe errors we can still be noisier about